### PR TITLE
New package: anbox-modules-git-0.0.0.20180908

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: srcpkgs/anbox-modules-git/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'anbox-modules-git'
+pkgname=anbox-modules-git
```
Comment (newbluemoon) [inline srcpkgs/anbox-modules-git/template]: I think `pkgname` should just be ‘anbox-modules’ without the ‘git’.

**Comment by Vaelatern** [comment]
Why do we want a git package here?

File: srcpkgs/anbox-modules-git/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+version=0.0.0.20180908
+revision=1
+_commit=27fd47e11ef6eef93738f8f3df3e42c88975544e
+wrksrc="${pkgname%-git}-${_commit}"
+conf_files="/etc/modules-load.d/anbox.conf"
+short_desc="Anbox kernel modules"
```
Comment (Vaelatern) [inline srcpkgs/anbox-modules-git/template]: Maybe `Kernel modules necessary for the Anbox Android container runtime` or something more descriptive?

**Comment by Vaelatern** [comment]
https://github.com/anbox/anbox-modules/issues/13

File: srcpkgs/anbox-modules-git/template (hunk 2 above)
Comment (CameronNemo) [reply srcpkgs/anbox-modules-git/template]: Sounds good.

**Comment by CameronNemo** [comment]
RIP this pull request. Perhaps we could just build ashmem and binder as modules for the regular kernel? They are upstream, the anbox modules are just for distros that do not build them (ie most distros).

**Comment by Hoshpak** [comment]
Looking at the kernel configuration it seems to me that these drivers are built exclusively for the Android platform and can't be built as modules so that doesn't seem to be a viable alternative either.

**Comment by sreehax** [comment]
uhh why not? I managed to build binder and ashmem as dkms modules

**Comment by CameronNemo** [comment]
@sreehax you built them out of tree using these sources I assume? The in-kernel versions cannot be built as modules according to Hoshpak.

**Pull request closed without merging**